layouts/page.njk

```diff
@@ -17,5 +17,9 @@
     {{ content }}
   {% endcall %}
 
-  {{ xGovukRelatedNavigation(related) if related.sections[0].items | length > 0 }}
+  {#- Related items may contain many sections or none -#}
+  {%- set sections = related.sections or [related] -%}
+  {{ xGovukRelatedNavigation({
+    sections: sections
+  }) if related.sections or related.items | length > 0 }}
```

Comment:
This block is repeated a lot, can it go into a partial somehow? 🤔

Reply:
Good idea.